src/components/App/ModalApp.tsx

```diff
@@ -226,6 +233,17 @@ class ModalApp extends Component<Props> {
     }
   }
 
+  setDecoupleFromAPIIfClientHasFeature = (
+    isValidEnterpriseFeature: boolean
+  ) => {
+    if (isValidEnterpriseFeature) {
```

Comment:
Maybe I am misunderstanding but I think `false` and `undefined` should be an invalid configuration. This code runs only if the integrator has set `decouple` (now changed to `useCustomizedApiRequests`) to a truthy value. If the feature is set to false or not defined on their account then they should get an invalid configuration error.

Reply:
Yeah after having another thought on this, I think handling any values other than `true | undefined` makes more sense here:
- If there's no `useCustomzedApiRequests` option set, use the default configuration (ignoring the callback options).
- If `useCustomizedApiRequests: true`, assert if the callback options are valid.
- If anything else: reports an invalid configuration error.

Reply:
Okay, I think I have this working as you described now.

On line 206/207 we get the value of `useCustomizedApiRequests` from their configuration options. Only continue with checking validity if truthy value is given, otherwise proceed as usual, ignoring callbacks.

If truthy:
- Check if authorized (features is enabled for their account and included in their token - line 246)
- If authorized, check that the callbacks are given and are functions, otherwise emit a `CustomApiException`
- If not authorized, (feature not included in their token or set to false) then emit `EnterpriseFeatureNotEnabledException`

Let me know if I misunderstood or missed something